### PR TITLE
fix(backing Image) Modify delete prompt

### PR DESCRIPTION
**Pull request by smallteeths**
https://github.com/longhorn/longhorn/issues/2506

image: wangsiye/longhorn-ui:f12341d

Signed-off-by: Siye Wang <siye@rancher.com>

### CODE REVIEW

**Review by shuo-wu — APPROVED** [review]

File: src/routes/backingImage/DiskStateMapDetail.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ const modal = ({
     switch (event.key) {
       case 'delete':
         confirm({
-          title: `Are you sure you want to delete backing iamge in ${record.disk} ?`,
+          title: `Are you sure to delete the images from the disk ${record.disk} ?`,
```
Comment (shuo-wu) [inline src/routes/backingImage/DiskStateMapDetail.js:34]: Singular `image` should be used here

**Review by innobead — APPROVED** [review]

File: src/routes/backingImage/DiskStateMapDetail.js (hunk 1 above)
Comment (innobead) [reply src/routes/backingImage/DiskStateMapDetail.js:34]: @smallteeths sorry, just merged. Please help fix this typo later on. Thanks.

Comment (smallteeths) [reply src/routes/backingImage/DiskStateMapDetail.js:34]: ok

Comment (smallteeths) [reply src/routes/backingImage/DiskStateMapDetail.js:34]: wu-shuo suggested changing `images` to `image file/files`

**Pull request merged**